src/components/LemmaInput.vue

```diff
@@ -51,31 +61,33 @@ export default {
 </script>
 
 <template>
-	<text-input
+	<cdx-field
 		class="wbl-snl-lemma-input"
-		:label="messages.getUnescaped( 'wikibaselexeme-newlexeme-lemma' )"
-		:placeholder="messages.getUnescaped(
-			'wikibaselexeme-newlexeme-lemma-placeholder-with-example',
-			exampleLemma
-		)"
-		name="lemma"
-		aria-required="true"
-		:error="error"
-		:value="modelValue"
-		@input="$emit( 'update:modelValue', $event )"
+		:status="error.status"
+		:messages="error.messages"
 	>
-		<template #suffix>
-			<required-asterisk />
+		<!-- eslint-disable vue/v-on-event-hyphenation -->
+		<cdx-text-input
+			:placeholder="messages.getUnescaped(
+				'wikibaselexeme-newlexeme-lemma-placeholder-with-example',
+				exampleLemma
+			)"
+			name="lemma"
+			aria-required="true"
+			:model-value="modelValue"
+			@update:modelValue="$emit( 'update:modelValue', $event )"
+		/>
+		<!-- eslint-enable -->
+		<template #label>
+			{{ messages.getUnescaped( 'wikibaselexeme-newlexeme-lemma' ) }}<required-asterisk />
 		</template>
-	</text-input>
+	</cdx-field>
 </template>
 
 <style lang="scss">
 @import '@wmde/wikit-tokens/variables';
```

Comment:
Should we not be removing this import? And replacing the SASS variable below with a CSS variable?

Reply:
In principle yes, but I thought this would be part of a separate task (T369510 if I’m not mistaken) and not included in this PR.

Reply:
eh. fair enough. I guess it will be some weeks before UX responds on that PR, so I'll end up rebasing it anyway.